provider/cmd/pulumi-gen-aws-native/supported-types.txt

```diff
@@ -111,6 +111,7 @@ AWS::EC2::NetworkInsightsPath
 AWS::EC2::PrefixList
 AWS::EC2::RouteTable
 AWS::EC2::SpotFleet
+AWS::EC2::Subnet
```

Comment:
Should we try and rewrite the ECS examples etc. to leverage the VPC and endpoint support?

Reply:
Yeah, but let's keep that separate from the update PR. We should probably make periodic update passes while existing resource support is still being added.

Reply:
yeah agreed. Not sure if this is sufficient anyway to replace awsx.vpc but worth taking a pass separately.